typescript/utils/src/altvm.ts

```diff
@@ -0,0 +1,657 @@
+import { MINIMUM_GAS } from './mingas.js';
+import { ProtocolType } from './types.js';
+
+// ### QUERY BASE ###
+export type ReqGetBalance = { address: string; denom: string };
+
+export type ReqGetTotalSupply = { denom: string };
```

Comment:
i think that numeric fields should be numbers instead of strings as it make it less error prone and less confusing when having to work on math operations as the string might be either the bigint represention or the decimal one. We should stick to using bigint for internal usage and string/number for representing the numbers in a human friendly way in logs or error messages. Otherwise a comment should state explicitly what the expected format is for the field